docs/pbc/all/usercentrics/integrate-usercentrics.md

```diff
@@ -83,7 +40,7 @@ All of the three approaches require you to get the setting ID for your store in
 
 {% endinfo_block %}
 
-#### Direct integration
+### Direct integration
 
 The direct integration requires quite a bit of manual work by developers.
```

Comment:
Not sure why we need to mention this. Would a `manual integration` be a better match for this way?

Reply:
Usercentrics calls this "direct implementation", and our engineers also prefer this term :)